packages/react/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
curious to learn about the purpose of the `packages/react`.

Apart from the RQ query client, what else will it contain?

Reply:
Nothing. Removed locally already xD.

If we want to share the query-client, we can probably do so through the API package.

Maybe we'll have some reusable hooks between platforms 🤔